api/v1/helpers/verbs/utils.js

```diff
@@ -237,9 +237,36 @@ function handleAssociations(reqObj, inst, props, method) {
   });
 } // handleAssociations
 
-function isMultipleAssociations(opts) {
-  const MULTIPLE_ASSOCIATIONS = 3;
-  return opts.attributes.length >= MULTIPLE_ASSOCIATIONS;
+/**
+ * When more than one association is required, Sequelize creates inner query
+ * but doesn't expose FK causing SQL failure.
+ *
+ * This method, in order to avoid Sequelize issue, will add all
+ * the foreign keys to the query when more than one association is required.
+ */
```

Comment:
Good solution.
The foreign key will still be included in the response in the corner case of multiple associations, correct? I think that's fine, but can you add a test to specify that that's the expected behavior?